partcad/src/partcad/runtime_python.py

```
@@ -61,18 +58,17 @@
         if venv is None:
-            self.venv = f"{runtime.sandbox}.default"
+            self.venv = f"{runtime.sandbox_dir}.default"
             self.venv_lock_name = f".{self.venv}.lock"
         else:
             self.venv = venv
-            self.venv_lock_name = f".{runtime.sandbox}.{venv}.lock"
+            self.venv_lock_name = f".{runtime.sandbox_dir}.{venv}.lock"
 
     async def __aenter__(self, *_args):
         await sync_threads.run_detached(self.runtime.venv_locks_lock.acquire)
         if not self.venv in self.runtime.venv_locks:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)